components/host-dashboard/AcceptRejectButtons.js

```diff
@@ -0,0 +1,68 @@
+import React, { Fragment, useState } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Mutation } from 'react-apollo';
+import gql from 'graphql-tag';
+
+import StyledButton from '../StyledButton';
+import AppRejectionReasonModal from './AppRejectionReasonModal';
+
+const ApproveCollectiveMutation = gql`
+  mutation approveCollective($id: Int!) {
+    approveCollective(id: $id) {
+      id
+      isActive
+    }
+  }
+`;
+
+const AcceptRejectButtons = ({ collective, host }) => {
+  const [showRejectionModal, setShowRejectionModal] = useState(false);
+  const [collectiveId, setcollectiveId] = useState(null);
```

Comment:
Why using a state here? Can't we rely directly on `collective.id`?

Reply:
I am using state to trigger the AppRejectionReasonModal.